config/300-clustertask.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
```

Comment:
What does this do?

Reply:
I just copied that from `task.yaml` but its left over from when these files were generated and should be removed. will take it out from both